hypervisor/vm.go

```diff
@@ -711,37 +715,29 @@ func (vm *Vm) GetIPAddrs() []string {
 	return ips
 }
 
-// sendGenericOperation queues a generic operation onto the VM's context if it
-// is in position to handle it.
-func (vm *Vm) sendGenericOperation(name string, op func(ctx *VmContext, result chan<- error), states ...string) <-chan error {
-	result := make(chan error, 1)
-
+func (vm *Vm) GenericOperation(name string, op func(ctx *VmContext, result chan<- error), states ...string) error {
```

Comment:
@laijs sure, now I just rename `sendGenericOperation()` to `handleGenericOperation()` (for what it really do), would this be fine?